Controllers/HostController.cs

```diff
@@ -139,6 +139,63 @@ public HttpResponseMessage CreateNewHost(Host host)
             }
             return Request.CreateResponse(HttpStatusCode.OK, $"NEW HOST CREATED SUCCESFULLY: {host.Name} {host.LastName}");
         }
+        // ===================================================================================================
+        [AllowAnonymous]
+        [HttpPost]
+        [Route("auth")]
```

Comment:
Methods attributes can be declared on this way which looks cleaner

```suggestion
 [HttpPost, AllowAnonymous, Route("auth")]
```

Reply:
Allright, thanks for the feedback!